webpack.config.js

```diff
@@ -16,7 +16,17 @@ module.exports = {
     },
 
     module: {
-        rules: [{ test: /\.(ts|js)x?$/, loader: 'babel-loader', exclude: /node_modules/ }],
+        rules: [
+            {
+                test: /\.(ts|js)x?$/,
+                loader: 'babel-loader',
+                exclude: /node_modules/
+            },
+            {
+                test: /\.css$/,
+                use: ['style-loader', 'css-loader']
+            },
+            {test: /\.ts$/, use: 'ts-loader'},],
     },
```

Comment:
you need file loader here as well. You should have pictures on homepage, it is the part of this task.